tools/place_name_resolver/resolver.go

```diff
@@ -263,16 +268,19 @@ func writeOutput(outCsvPath string, tinfo *tableInfo) error {
 	return nil
 }
 
-func resolvePlacesByName(inCsvPath, outCsvPath string, p2d PlaceId2Dcid, mapCli MapsClient) error {
+func resolvePlacesByName(inCsvPath, outCsvPath string, generatePlaceID bool, p2d PlaceId2Dcid, mapCli MapsClient) error {
 	tinfo, err := buildTableInfo(inCsvPath)
 	if err != nil {
 		return err
 	}
-	placeIdToDcid, err := loadPlaceIdToDcidMap(p2d)
-	if err != nil {
-		return err
+	placeId2Dcid := &map[string]string{}
```

Comment:
map is "pointer" like, can just do "map[string]string{}"

Reply:
So its okay fro loadPlaceIdToDcidMap() to return a map, rather than a map pointer?  Wei had in the past suggested passing/returning pointers...

Reply:
Map itself is okay: https://stackoverflow.com/questions/40680981/are-maps-passed-by-value-or-by-reference-in-go

Reply:
Thanks, changed all pointers to refs.